browse/controllers/home_page/__init__.py

```diff
@@ -39,14 +40,12 @@ def get_home_page() -> Response:
 
 def _get_document_count() -> Optional[int]:
 
-    try:
-        # check DB for document count first
+    try: # check DB for document count first
         return get_document_count()  # type: ignore
     except Exception as ex:
-        logger.warning(f'Error getting document count from DB: {ex}')
+        logger.warning('Error getting document count from DB', ex)
```

Comment:
If not using f-strings, wouldn't this still need to be: 

```suggestion
        logger.warning('Error getting document count from DB: %s', ex)
```

Reply:
Yes, and it is suggested to not use f-strings in logging output so the string substitution is skipped if the logging message is not needed.